Added headroom menu

asharonbaltazar opened the pull request:
Issue #281, implemented a headroom menu with a lightweight npm package (svelte-headroom). The header obviously broke some styling, which I did my best to fix. I also implemented an event to close the menu (if it's open) when scrolling. Please have a look. 

Edit : noticed more styling tweaks needed. Happy to discuss an orderly approach rather than haphazardly scattering `z-10`s :stuck_out_tongue_winking_eye:

**Next move:**
Based on this restyling, the next issue to focus on is adding a small redesign to the menu, particularly the padding. I believe the whitespace at the top of the page balanced it out. Have a look here:
![image](https://user-images.githubusercontent.com/58940073/93120370-e408f980-f688-11ea-967d-132b9613ee35.png)

csell5 commented:
At first glance, we need to rework that top nav's overall layout. As it's not taking up the whole width of the page.

![image](https://user-images.githubusercontent.com/772569/93222740-e2950b00-f734-11ea-8ba3-5e672a525816.png)

You already mentioned the padding. I'd just make it equal on both the top and bottom so the logo is centered vertically.

And I'd love to jump on and chat more if you'd like. I'd say we just throw a session on that.us and chat.

asharonbaltazar commented:
Sure, I'm free in about 15 mins. How does that sound?

csell5 commented:
well I'm streaming now. How about 3pm central? Not sure where you're located.

asharonbaltazar commented:
I'm in central too. 3 pm sounds good.

csell5 commented:
> I'm in central too. 3 pm sounds good.

@asharonbaltazar Perfect. We can meet here: https://that.us/sessions/QzDQlvIeSIWj0okaNDMH

csell5 commented on src/routes/Home.svelte:
```diff
@@ -44,7 +44,7 @@
 -->
 
-<div class="max-w-screen-xl mx-auto">
+<div class="relative max-w-screen-xl mx-auto z-10">
```
I know you made a comment about the zindex but I'm not sure how we'd get around that.

asharonbaltazar replied:
What should I do about this?

csell5 replied:
we talked about this yesterday.

asharonbaltazar replied:
I don't think there's any way to get around this right now. Removing those classes causes the navbar to disappear

csell5 commented:
Looks like you might still be working through this.. just re-request when you're ready.

asharonbaltazar commented:
Well, other than your comment above and during a sight margin issue, I think it's ready. 

I do, however, want the menu to sort "stand out" from the rest of the page's white background. Any suggestions for this?

csell5 commented:
I still show the nav background not taking the whole screen 
![image](https://user-images.githubusercontent.com/772569/93361406-7553a980-f80a-11ea-96ea-d808b62f4af9.png)

csell5 commented:
I'm not sure I was totally clear in what I mean.. I was referring to the left side of the nav bar.

![image](https://user-images.githubusercontent.com/772569/93371536-23b21b80-f818-11ea-8cbe-b8061b2cc9d5.png)

asharonbaltazar commented:
Other than this, is there anything else that needs work?

asharonbaltazar commented:
> I'm not sure I was totally clear in what I mean.. I was referring to the left side of the nav bar.

Addressed. Works fine on the emulator. Lemme know how it's for you.

csell5 commented:
I have to apologize, I don't seem to be expressing it correctly. Yes that does work, but we just needed the background to extend the whole way, not the contents of the nav bar. Now when you're on a bigger screen ( like me ) that nav is outside of the main content block.

![image](https://user-images.githubusercontent.com/772569/93474433-f1a6c500-f8bc-11ea-91c3-03dc59f2f735.png)

So essentially the white background, should take the whole width. But the contents of the nav, should be visually centered like they were before.

asharonbaltazar commented:
Can we just do a `flex justify-center` when on super large screens?

asharonbaltazar commented:
I'm having trouble launching the project on the localhost. Unable to figure what's the issue.

asharonbaltazar commented:
FIXED

csell5 commented:
Looking pretty hottt but it only lays out correctly on the home page. Check out https://thatus-kt2ybdv8u.vercel.app/members and https://thatus-kt2ybdv8u.vercel.app/partners.

This is in part as I should really have a dedicated layout for the "outside pages" vs the "inside pages" but if I remember correctly (??) those pages are pulling in the nav directly rather than a standard layout which has the nav.

asharonbaltazar commented:
So if those are fixed, then what else is there to do?

asharonbaltazar commented:
Okay, so these commits simply address the navbar. However, there are NEW issues (surprise, surprise). I think the Members page has a z-index problem. 

If you want, you can merge this, and start addressing those issues in separate places.

csell5 commented:
So I see. Rebasing it now. I'll review deeper here. I'm also working on a new `above the fold` layout that might work better for members and partners anyways.

csell5 commented:
@asharonbaltazar nice job! thank you!

asharonbaltazar commented:
I'll be eyeing the issues tab for more work. Thanks for the opportunity.
